Makefile

```diff
@@ -90,14 +70,20 @@ proto:
 # Installs dependencies for building the user interface.
 setup-ui-libs:
 	cd ui; npm i; cd ..
+	cd provider-ui; npm i; cd ..
 
-# Runs the UI interface on your local machine.
+# Runs the Meshery UI interface on your local machine.
 run-ui-dev:
 	cd ui; npm run dev; cd ..
 
-# Builds the user interface on your local machine.
+# Runs the Provider UI interface on your local machine.
+run-provider-ui-dev:
+	cd provider-ui; npm run dev; cd ..
```

Comment:
@girishranganathan the locally running provider ui and the regular ui collide on port 3000.

Reply:
I have changed the port a different one now.